docs/stencil-docs/deploying-a-theme/theme-best-practices.md

```diff
@@ -0,0 +1,174 @@
+
+**Good score:**
+The optimal LCP score is 2.5 seconds or faster. 
+
+</div>
+</div>
+</div>
+
+To improve LCP:
+* Make images smaller or remove anything preventing a quick download.
+* Avoid JavaScript or external scripts to load images.
+* Use an image CDN
+* Optimize your server
+
+### First Input Delay
+FID is the metric that measures the delay users experience when interacting with a page. It is the time it takes for the site to respond when a user clicks a link, selects a button, taps on the screen, etc.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal FID score is 100 milliseconds or less. 
+
+</div>
+</div>
+</div>
+
+To improve FID:
+* Reduce the effect of third-party code
+* Reduce JavaScript execution time
+* Reduce duration of your longest task
+
+Lighthouse does not display FID since it can only be measured in the field. Real user interaction is needed in order to measure the response delay. Instead, Lighthouse displays Total Blocking Time (TBT) which also captures a user's interactivity. Improvements to TBT will also improve your FID score.
+
+A good TBT score is 300 milliseconds or less.
+
+### Cumulative Layout Shift (CLS)
+
+CLS is the metric that measures visual stability. This metric measures how often users experience unexpected layout shifts. A layout shift is when a visible element changes its position from one rendered frame to the next. For example, a layout shift can occur when a user is viewing a page and a video moves on top of the home carousel. These unexpected movements can result in an unpleasant experience for the user.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal CLS score is 0.1 seconds or less. 
+
+</div>
+</div>
+</div>
+
+To improve CLS:
+* Add size attributes to elements
+* Reserve space for injected content
+* Avoid inserting ads or banners dynamically
+* Ensure to load critical CSS before content
+
+### Optimization example
+
+In this example, the performance score is currently 47. The LCP score is 6.2s, the TBT score is 410 ms, and the CLS score is 1.137 which are all less than optional scores.
+
+![Performance Metrics](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/performnce-example-1.png "Performance Metrics")
+
+To improve performance, let's start by removing usused JavaScript. Lighthouse and the **Coverage** tab offers suggestions of unused JavaScript code that can be removed. The red section of the bar is unused bytes. The green section is used bytes.
+
+![Coverage tab](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/performnce-example-2.png "Coverage tab")
+
+In addition to removing unused bytes, you can minify, compress, and refractor CSS/JavaScript files. This includes removing line breaks, whitespace, and comments.
+
+**Minify example**
+
+```json
```

Comment:
I think `css` will work here as well

Reply:
Yes, this example comes from custom.css. 
I changed ```json to ```css.